apps/website/.vuepress/theme/layouts/Layout.vue

```diff
@@ -20,7 +20,12 @@
       <Sidebar :items="sidebarItems" :isSidebarOpen="isSidebarOpen" @isSidebarOpenChange="toggleSidebar()" />
       <div id="content-area" tabindex="-1" :class="{ 'content-area': true, 'home-page': $page.frontmatter.home }">
         <Home v-if="$page.frontmatter.home" class="make-it-scrollable" />
-        <div v-if="is404" class="make-it-scrollable" cds-layout="horizontal gap:md align:horizontal-center">
+        <div
+          v-if="is404"
+          id="page-not-found"
+          class="make-it-scrollable"
+          cds-layout="horizontal gap:md align:horizontal-center"
```

Comment:
Lets use the `p-t@sm:md p-t:sm` token here. It may need to be adjusted for the t-shirt sizes used.

Reply:
much cleaner!